tests/admin-unlock/main.yml

```diff
@@ -0,0 +1,316 @@
+#   Verify overlayfs is created after running ostree admin unlock
+#     (use mount command verify presence of overlayfs mount)
+#   Verify ostree admin status indicates "Unlocked: development" after running
+#     ostree admin unlock
+#   Verify that changes to deployment after ostree admin unlock do not persist
+#     through reboot.
+#   Verify overlayfs is removed (use mount command verify absence of overlayfs
+#       mount)
+#   Verify ostree admin status does not show locked deployment
+#   Verify rpms can be installed and uninstalled after ostree admin unlock
+#     --hotfix and changes persist through reboot.
+#   Verify ostree admin status indicates "Unlocked: hotfix"
+#   Verify a clone of the current deployment is added to the deployment
+#   Use rpm-ostree status to verify there are two deployments with the same
+#     commit ID
+#   Verify overlayfs is created (use mount command verify presence of overlayfs
+#      mount)
+#   Verify rpms can still be installed and uninstalled after reboot
+#   Verify ostree admin unlock --hotfix overwrites any other deployment already
+#      on the system
+#   Verify rollback between hotfixed deployment and regular deployment
+# NOT TESTED:
+#   Verify upgrade from hotfixed deployment
+#     Upgrade should be successful
+#     Packages installed in previous unlocked deployment should not be present
+#       in the upgraded deployment
+#
+# Negative Testing
+#   Verify rpms cannot be installed without ostree admin unlock
+#   Run ostree admin unlock twice.  Verify that an error message specifying
+#     that the deployment is already in an unlocked state: development
+#   Run ostree admin unlock, then ostree admin unlock --hotfix.  Verify that
+#     an error message specifying that the deployment is already in an unlocked
+#     state: development
+#   Run ostree admin unlock --hotfix twice.  Verify that an error message
+#     specifying that the deployment is already in an unlocked state: hotfix
+#   Run ostree admin unlock --hotfix then ostree admin unlock.  Verify that an
+#     error message specifying that the deployment is already in an unlocked
+#     state: hotfix
+
+- name: Ostree Admin Unlock - Setup
+  hosts: all
+  become: yes
+
+  tags:
+    - setup
+
+  vars_files:
+    - vars.yml
+
+  pre_tasks:
+    - name: Verify all
+      fail:
+        msg: "Required variables are not defined.  Check vars.yml file."
+      when: g_rpm_name is undefined or
+            g_rpm_name_2 is undefined
+
+  roles:
+    # Subscribe if the system is RHEL
+    - role: redhat_subscription
+      when: ansible_distribution == 'RedHat'
+      tags:
+        - redhat_subscription
+
+- name: Ostree Admin Unlock - Add/Remove Package - Non-persistence
+  hosts: all
+  become: yes
+
+  tags:
+    - unlock_add_remove_package_non_persistence
+
+  vars_files:
+    - vars.yml
+
+  pre_tasks:
+    - name: set rpm_name
+      set_fact:
+        rpm_name: "{{ g_rpm_name }}"
```

Comment:
Rather than setting the `rpm_name` here, perhaps pass it in as an argument for role that requires it?

Since that method is used when installing multiple RPMs, it might make sense to use that method everywhere.